packages/api-plugin/src/generators/trpc-api-generator/files/src/constants/cors.ts

```diff
@@ -0,0 +1,12 @@
+import { envVars } from './envVars'
+
+/**
+ * Allowlist of regexes of websites that will accept cors headers
+ * When we open source we can make this an env variable
+ */
+export const corsWhitelist = [
```

Comment:
it would be better to call this an allowlist/(blocklist/denylist) instead of whitelist/blacklist